shell/dialog/MoveNamespaceDialog.vue

```diff
@@ -0,0 +1,159 @@
+<script>
+import { mapGetters } from 'vuex';
+import { Card } from '@components/Card';
+import AsyncButton from '@shell/components/AsyncButton';
+import LabeledSelect from '@shell/components/form/LabeledSelect';
+import { MANAGEMENT } from '@shell/config/types';
+import { PROJECT } from '@shell/config/labels-annotations';
+
+export default {
+  emits: ['close'],
+
+  components: {
+    AsyncButton, Card, LabeledSelect
+  },
+
+  props: {
+    resources: {
+      type:    Array,
+      default: () => []
+    },
+    movingCb: {
+      type:    Function,
+      default: () => {}
+    },
+    // TODO: handle console warns if we remove this prop...
```

Comment:
nitpick: We should have github issues to reference in each TODO. Without an issue, the odds are low that this work will ever get prioritized.